extensions/docbook-snippets/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to your VS Code Extension
+
+## What's in the folder
+
+* This folder contains all of the files necessary for your extension.
+* `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
+* `snippets/snippets.code-snippets` - the file containing all snippets.
+
+## Get up and running straight away
```

Comment:
What does _Get up and running_ mean? Build an extension? Install it? Run it? Something else?

Also, the hed sounds a bit colloquial to me